### PR TITLE
adding option to re-create search with cli

### PR DESCRIPTION
**Pull request by deadc0de6**
adding a client option (-r --reindex) to force creation of the index. This is useful when changing/adding pages directly on disk

### CODE REVIEW

**Comment by mblayman** [comment]
Thanks for the contribution! I'll add one or two comments to the diff, then I'll be happy to merge.

* Feel free to add your name to the AUTHORS file. It's alphabetic by first name.
* There's a pep8 error (missing a newline). You can check the Travis build for details or run pep8 locally to see it yourself.

File: markwiki/views/core.py [hunk 1]
```diff
@@ -182,3 +182,10 @@ def search():
     results = app.search_engine.search(user_query)
     return render_template('search.html', user_query=user_query,
                            results=results)
+
+@app.route('/reindex/')
+def reindex():
+    results = app.search_engine.create_index(app.config['WIKI_PATH'])
+    flash('Search index re-created', 'info')
+    '''Display the MarkWiki main page.'''
```
Comment (mblayman) [inline markwiki/views/core.py]: I don't think this comment is really helping and it's a docstring in the middle of an implementation block which is a bit odd. Please remove this line.

**Comment by deadc0de6** [comment]
fixed pep8 and the weird comment.

**Comment by mblayman** [comment]
Looks good. Thanks again!

**Pull request merged**